frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideOutBranchAction.java

```diff
@@ -104,7 +104,7 @@ private void deleteBranchIfRequired(AnActionEvent anActionEvent, String branchNa
     var project = getProject(anActionEvent);
     var slidOutBranchIsCurrent = getCurrentBranchNameIfManaged(anActionEvent)
         .map(b -> b.equals(branchName))
-        .getOrElse(true);
+        .getOrElse(false);
```

Comment:
Whoops

Reply:
With the old impl, we were assuming that slid-out branch is current when the current branch was unmanaged ☹️

Reply:
mea culpa